avocado-setup.py

```diff
@@ -449,16 +449,25 @@ def parse_test_config(test_config_file, avocado_bin, enable_kvm):
         for section in [dist, major, minor, minor_env]:
             if NORUNTESTFILE.has_section(section):
                 norun_tests.extend(NORUNTESTFILE.get(section, 'tests').split(','))
+        norun_tests = list(filter(None, norun_tests))
 
         with open(test_config_file, 'r') as fp:
             test_config_contents = fp.read()
         test_list = []
         mux_flag = 0
         for line in test_config_contents.splitlines():
+            norun_flag = False
             test_dic = {}
-            if line in norun_tests:
-                continue
             if line.startswith("#") or not line:
+                norun_flag = True
```

Comment:
norun_flag is set always True ?

Reply:
@sathnaga in line 459, it is defaulted to False.

Reply:
```for norun_test in norun_tests:
    if norun_test.endswith('*') and line.startswith(norun_test[:-1]) and not line.startswith("#"):
        norun_flag = True
        break```

Reply:
do we need other checks?

Reply:
`line in norun_tests` is surely needed.
And I thought code was more readable if there were 3 if cases, each covering one of:
1. empty line or comment
2. exact match
3. * for generic match